src/main/java/com/conveyal/datatools/manager/models/Deployment.java

```diff
@@ -225,7 +221,7 @@ public Date retrieveLastDeployed () {
         return ret;
     }
 
-    /** Create a single-agency (testing) deployment for the given feed source */
+    /** Create a single-agency deployment for the given feed source */
```

Comment:
This deployment type is primarily used for testing out a new GTFS feed in an OTP deployment. I think this comment should remain unchanged (or can be clarified further if it was confusing in its previous iteration).

Reply:
Reverted to previous comment.